mlir_graphblas/src/test/GraphBLAS/invalid_matrix_multiply.mlir

```diff
@@ -0,0 +1,175 @@
+// RUN: graphblas-opt %s -split-input-file -verify-diagnostics
+
+#CSR64 = #sparse_tensor.encoding<{
+  dimLevelType = [ "dense", "compressed" ],
+  dimOrdering = affine_map<(i,j) -> (i,j)>,
+  pointerBitWidth = 64,
+  indexBitWidth = 64
+}>
+
+module {
+    func @matrix_multiply_wrapper(%argA: tensor<2x3xi64>, %argB: tensor<3x2xi64, #CSR64>) -> tensor<2x2xi64, #CSR64> {
```

Comment:
I don't want to hold up this PR, but in a future PR we should verify that A is CSR, B is CSC, and output is CSR.

I also want to continue discussions about separating transpose into two functions, but I don't want to delay this PR for that.

Reply:
@jim22k, I made the changes to check this in https://github.com/metagraph-dev/mlir-graphblas/commit/97fcab915d9a97ecbf22820bd00f7faeeaca293e. I added and updated tests as well.